enterprise_subsidy/apps/transaction/management/commands/write_reversals_from_enterprise_unenrollments.py

```diff
@@ -0,0 +1,182 @@
+            if run.get('key') == enrollment_course_run_key:
+                course_start_date = run.get('start')
+                break
+
+        if not course_start_date:
+            logger.warning(
+                f"No course start date found for course run: {enrollment_course_run_key}. "
+                "Unable to determine refundability."
+            )
+            return False
+
+        # https://2u-internal.atlassian.net/browse/ENT-6825
+        # OCM course refundability is defined as True IFF:
+        # ie MAX(enterprise enrollment created at, course start date) + 14 days > unenrolled_at date
+        enrollment_created_at = enterprise_course_enrollment.get("created")
+        enrollment_unenrolled_at = enterprise_course_enrollment.get("unenrolled_at")
+        enrollment_created_datetime = datetime.strptime(
+            enrollment_created_at, "%Y-%m-%dT%H:%M:%SZ"
+        )
+        course_start_datetime = datetime.strptime(
+            course_start_date, "%Y-%m-%dT%H:%M:%SZ"
+        )
+        enrollment_unenrolled_at_datetime = datetime.strptime(
+            enrollment_unenrolled_at, "%Y-%m-%dT%H:%M:%SZ"
+        )
+        refund_cutoff_date = max(course_start_datetime, enrollment_created_datetime) + timedelta(days=14)
+
+        if refund_cutoff_date > enrollment_unenrolled_at_datetime:
+            logger.info(
+                f"Course run: {enrollment_course_run_key} is refundable for enterprise customer user: "
+                f"{enterprise_course_enrollment.get('enterprise_customer_user')}. Writing Reversal record."
+            )
+            return True
+        else:
+            logger.info(
+                f"Unenrollment from course: {enrollment_course_run_key} by user: "
+                f"{enterprise_course_enrollment.get('enterprise_customer_user')} is not refundable."
+            )
+            return False
+
+    def handle(self, *args, **options):
+        """
+        Fetch enterprise enrollment objects that have been unenrolled within the last 24 hours and write Transaction
+        Reversals where appropriate for each unenrollment.
+        """
+        logger.info("Updating and writing Transaction Reversals from recent Enterprise unenrollments data")
+        recent_unenrollments = EnterpriseApiClient().fetch_recent_unenrollments()
+        logger.info(f"Found {len(recent_unenrollments)} recent Enterprise unenrollments")
+
+        reversals_processed = 0
+        enterprise_catalog_api_client = EnterpriseCatalogApiClient()
+        fetched_content_metadata = {}
+        for unenrollment in recent_unenrollments:
+            fulfillment_uuid = unenrollment.get("uuid")
+            enterprise_course_enrollment = unenrollment.get("enterprise_course_enrollment")
+            enrollment_course_run_key = enterprise_course_enrollment.get("course_id")
+            enrollment_unenrolled_at = enterprise_course_enrollment.get("unenrolled_at")
+            try:
```

Comment:
Nit: can you add an `else` block for this `try` to log a message like "Reversal found for unenrollment of enterprise fulfillment: {fulfillment_uuid}. Skipping."